packages/components/src/utils/get-focusable-children.ts

```diff
@@ -23,6 +23,6 @@ const focusDisablingSelector = `:where(${[
   'details:not([open]) > *:not(details > summary:first-of-type) *',
 ].join(',')})`;
 
-export const isFocusable = (element: Element) => {
-  return element?.matches(focusableSelector) && !element?.matches(focusDisablingSelector);
+export const getFocusableChildren = (element: Element): NodeListOf<HTMLElement> => {
+  return element.querySelectorAll(`& > ${focusableSelector}:not(${focusDisablingSelector})`);
```

Comment:
I think there is a problem in the `focusableSelector`, because `tabindex` with any negative value will make an element unfocusable.
Something like `:not([tabindex^="-"])` could maybe work.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex